src/views/Board.py

```diff
@@ -198,6 +188,10 @@ def is_valid_move(self, from_position: Position, to_position: Position | None):
         ):
             return False
 
+        if piece.animal == Animal.HOUND:
+            if to_position.x < from_position.x:
+                return False
+
```

Comment:
Acho q pode colocar isso na mesma sentença
```suggestion
        if piece.animal == Animal.HOUND and to_position.x < from_position.x:
            return False

```